### ISSUE TITLE
Accessibility Issue - Calorie Counter Exercise

### ISSUE THREAD

**Issue by neirobic** [freeCodeCamp/freeCodeCamp#62889]
### Describe the Issue

The exercise topic will be very triggering to some people in the process of eating disorder recovery. Issue can be solved by changing the exercise framing, e.g. turning into a "Make an Envelope Budgeter" exercise by making some small changes to the form labels and exercise instructions. 

### Affected Page

https://www.freecodecamp.org/learn/full-stack-developer/workshop-calorie-counter/step-1

### Your code

```



```

### Expected behavior

--

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

Re: Eating disorders and accessibility: here's a page on eating disorders at work from a non-profit called Beat Eating Disorders. See under "How you can make a difference in your workplace", "Avoid talking about (...) calorie counting" https://www.beateatingdisorders.org.uk/get-information-and-support/support-someone-else/worried-about-someone-at-work/

Re: Envelope budgeting - it's a kind of budgeting system where you assign specific amounts of your total available money into categories ("envelopes"), then spend from them - https://en.wikipedia.org/wiki/Envelope_system

**Comment by raisedadead** [comment]
cc @freeCodeCamp/curriculum

**Comment by jdwilkin4** [comment]
Thank you for opening up this issue.

We will have one of the curriculum team members work on updating the app 👍🏾

**Comment by jdwilkin4** [comment]
As I was working through the original project and making updates, I noticed that there were more changes that should be implemented. I am going to create a separate issue in the curriculum expansion repo for more detailed instructions on the changes needed. 

As a result, I am going to close this issue in favor of this newer issue for a envelope budget app 👍🏾
